pkg/config/config_test.go

```diff
@@ -155,6 +157,61 @@ func TestNewConfig(t *testing.T) {
 	})
 }
 
+func Test_LoadConfig(t *testing.T) {
+	t.Run("valid-url", func(t *testing.T) {
+		c, err := loadConfig("https://raw.githubusercontent.com/get-woke/woke/main/example.yaml")
+		assert.NoError(t, err)
+		assert.NotNil(t, c)
+		// delete downloaded file after successful test
+		os.Remove("downloadedRules.yaml")
```

Comment:
I would suggest using `t.Cleanup` to handle cleanup like this https://pkg.go.dev/testing#T.Cleanup

Reply:
N/A after deleting download file functionality